pip/tests/test_qsharp.py

```diff
@@ -373,3 +375,77 @@ def test_target_profile_from_str_match_enum_values() -> None:
     assert qsharp.TargetProfile.from_str(str_value) == target_profile
     with pytest.raises(ValueError):
         qsharp.TargetProfile.from_str("Invalid")
+
+
+def test_callables_exposed_into_env() -> None:
```

Comment:
Could you break this out into a few different test cases?

Reply:
Will do!